.github/workflows/helm-chart.yml

```diff
@@ -22,35 +25,52 @@ jobs:
 
       - name: Configure git
         run: |
-          git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
 
       - name: Install helm
         uses: azure/setup-helm@v4
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
+      - name: Install helm-docs
+        env:
+          HELM_DOCS_VERSION: "1.14.2"
+        run: |
+          cd /tmp
+          wget https://github.com/norwoodj/helm-docs/releases/download/v"${HELM_DOCS_VERSION}"/helm-docs_"${HELM_DOCS_VERSION}"_Linux_x86_64.tar.gz
+          tar -xvf helm-docs_"${HELM_DOCS_VERSION}"_Linux_x86_64.tar.gz
+          sudo mv helm-docs /usr/local/bin
+
       - name: Set helm chart appVersion from Version.props
         run: |
           export VERSION_PROPS=$(awk -F'[<>]' '/VersionPrefix/{print $3}' Version.props)
           sed -i -e 's#Version.props#"'${VERSION_PROPS}'"#g' charts/garnet/Chart.yaml
 
-      - name: Helm lint and package
+      - name: Helm lint, helm-docs and helm package
         run: |
           mkdir .cr-release-packages
           for chart in $(find charts -mindepth 1 -maxdepth 1 -type d); do
             if [ -z "${chart:-}" ]; then
               break
             fi
             helm lint "${chart}"
+            helm-docs --document-dependency-values --chart-search-root "${chart}"
             helm package "${chart}" --dependency-update --destination .cr-release-packages
           done
 
+      - name: Git commit charts/garnet/README.md and push to main
```

Comment:
The main branch is protected, so all changes need to be made as a PR. Can the CI offer this change as a PR?

Reply:
@badrishc Indeed, the point is important. Apparently it is possible with the [gh](https://cli.github.com/manual/gh_pr_create) command to easily create a PR. I have to work on it.

Reply:
Or perhaps use [this github action](https://github.com/marketplace/actions/create-pr-action).

Reply:
This one has 2.2k stars: https://github.com/marketplace/actions/create-pull-request